### PR TITLE
Nimbus desktop toggle setting

### PR DESCRIPTION
**Pull request by kounkou**
### Description

This PR adds the Advanced setting toggle for Nimbus enabling.
It fixes [13204](https://github.com/status-im/status-desktop/issues/13204)

This PR is linked to : https://github.com/status-im/status-go/pull/4254

### Affected areas

- Settings/Advanced pannel

### Screenshot of functionality (including design for comparison)

[Screencast from 2024-01-18 01:06:00 PM.webm](https://github.com/status-im/status-desktop/assets/2589171/1f62b7bf-908f-457d-8fe7-9856a96156f8)

### CODE REVIEW

**Comment by kounkou** [comment]
So it turned out, @Vitaly already worked on the backend of the UI. So I'll revise my PR and connect the UI only to what was done in the PR https://github.com/status-im/status-go/pull/4254/files
Expectation should be that I only send a PR for the UI part.
For the sentence/labels changes, please let me know what reflects better the feature then I can make an update after above mentioned task.

**Review by vitvly — APPROVED** [review]
As much as i can understand, looks good! Thank you @kounkou !

**Review by MishkaRogachev — APPROVED** [review]
Looks good!

File: ui/app/AppLayouts/Profile/views/AdvancedView.qml [hunk 1]
```diff
@@ -81,6 +81,17 @@ SettingsContentBase {
                 }
             }
 
+            StatusSettingsLineButton {
+                anchors.leftMargin: 0
+                anchors.rightMargin: 0
+                text: qsTr("Verify all web3 data as a light client")
```
Comment (kounkou) [inline ui/app/AppLayouts/Profile/views/AdvancedView.qml:87]: Reminder to change the sentence from 

> Verify all web3 data as a light client

to

> Mainnet data verified by Nimbus

**Pull request merged**